### PR TITLE
Update rubygem-foreman_remote_execution to 1.8.0 (Foreman 1.22)

### PR DESCRIPTION
**Pull request by iNecas**
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [x] 1.22
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/plugins/rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec [hunk 1]
```diff
@@ -7,18 +7,24 @@
 
 Summary: SSH remote execution provider for Foreman smart proxy
 Name: rubygem-%{gem_name}
-Version: 0.2.0
+Version: 0.2.1
```
Comment (ehelms) [inline packages/plugins/rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec]: Can you reset the release to 1? Thats the test that is failing.

Comment (iNecas) [reply packages/plugins/rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec]: aye aye captain. updated

**Comment by iNecas** [comment]
Hi @theforeman/packaging, mind merging?

**Comment by iNecas** [comment]
@theforeman/packaging it's still doesn't seem to be in 1.22 repos https://yum.theforeman.org/plugins/1.22/el7/x86_64/, anyone with koji voodoo doll for the rescue?

**Comment by ekohl** [comment]
Our promotion pipeline has a regression that broke promotion. https://ci.theforeman.org/job/foreman-plugins-rpm-pipeline/

**Comment by adamruzicka** [comment]
Any updates on this?

**Comment by ekohl** [comment]
Yes, on the 17th I fixed the pipeline but forgot to comment it here.

**Pull request merged**